src/store/actions.ts

```diff
@@ -105,6 +105,38 @@ class Actions {
     dispatch('updateDataset');
   }
 
+  async selectRightDataset(
+    { commit, state }: ActionContext<VQBState, any>,
+    { rightPipelineLabel }: { rightPipelineLabel: string },
+  ) {
+    if (rightPipelineLabel != '') {
+      let activePipeline: Pipeline = [];
+
+      if (rightPipelineLabel in state.pipelines) {
+        activePipeline = state.pipelines[rightPipelineLabel];
+      } else {
+        activePipeline = [
+          {
+            name: 'domain',
+            domain: rightPipelineLabel,
+          },
+        ];
+      }
+
+      const response = await state.backendService.executePipeline(
+        activePipeline,
+        state.pipelines,
+        1,
+        0,
+      );
+      if (response.data) {
+        commit('setRightDataset', {
+          rightDataset: addLocalUniquesToDataset(response.data),
+        });
+      }
```

Comment:
`addLocalUniquesToDataset` have the purpose of creating a list of unique values for each column.
This is useful to display these values for filtering, in the column menu.

As we don't display the right dataset in a data table, nor want to filter it, I think you can skip this step.